disdrodb/utils/compression.py

```diff
@@ -0,0 +1,202 @@
+import bz2
+import glob
+import gzip
+import os
+import shutil
+import tempfile
+import zipfile
+from typing import Optional
+
+from ..api.checks import check_disdrodb_dir
+
+
+def _unzip_file(file_path: str, dest_path: str) -> None:
+    """Unzip a file into a folder
+    Parameters
+    ----------
+    file_path : str
+        Path of the file to unzip
+    dest_path : str
+        Path of the destination folder
+    """
+
+    with zipfile.ZipFile(file_path, "r") as zip_ref:
+        zip_ref.extractall(dest_path)

```

Comment:
@sphamba can you just test this function please?

Reply:
done